src/packages/cli/src/__tests__/fixtures/schema-db-out-of-sync/schema.prisma

```diff
@@ -8,7 +8,7 @@ datasource db {
   url      = "file:dev.db"
 }
 
-model Post {
+model NewPost {
```

Comment:
This was my hack to force desync. I copied over the `dev.db` from another fixture and then made changes to the prisma schema...

I think this is hard to maintain because we cannot "see" the state of the sqlite db?

Basically I want to be able to edit sqlite db as easily as I edit the prisma schema... :)

Reply:
Haha I highly recommend Table Plus for this :) I'm using that all the time to edit the dev.db files.

Reply:
Cool, added a tip about this in the contributing guide.